test/Unit.hs

```diff
@@ -100,5 +101,6 @@ prettyConst =
         )
     ]
  where
+  mkOp c t1 t2 = TApp (TApp (TConst c) t1) t2
```

Comment:
Let's fix the examples to use `mkOp'` instead :slightly_smiling_face:

Reply:
Oops, I forgot to remove the `mkOp'` definition, oh well let's use it, in case it is useful elsewhere.